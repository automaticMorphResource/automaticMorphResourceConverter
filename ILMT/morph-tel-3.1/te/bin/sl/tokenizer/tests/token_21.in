ఇక్కడ వన్యప్రాణులన్నీ జూలోనో, పార్కులోనో బంధించినట్లుగా కాక.. స్వేచ్ఛగా తిరుగుతూ ఉంటాయి. 
