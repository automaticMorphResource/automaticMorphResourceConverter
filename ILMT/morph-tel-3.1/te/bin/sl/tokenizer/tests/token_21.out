<Sentence id="1">
1	ఇక్కడ	unk
2	వన్యప్రాణులన్నీ	unk
3	జూలోనో	unk
4	,	unk
5	పార్కులోనో	unk
6	బంధించినట్లుగా	unk
7	కాక..	unk
8	స్వేచ్ఛగా	unk
9	తిరుగుతూ	unk
10	ఉంటాయి	unk
11	.	unk
</Sentence>

